src/protocols/secure_channel/CASESession.cpp

```diff
@@ -1820,7 +1997,7 @@ CHIP_ERROR CASESession::OnMessageReceived(ExchangeContext * ec, const PayloadHea
         switch (static_cast<Protocols::SecureChannel::MsgType>(payloadHeader.GetMessageType()))
         {
         case Protocols::SecureChannel::MsgType::CASE_Sigma3:
-            err = HandleSigma3(std::move(msg));
+            err = HandleSigma3a(std::move(msg));
```

Comment:
Rather than a, b, c, we could try to give meaningful names such as HandleSigma3IncomingMessage, HandleSigma3BackgroundCrypto, HandleSigma3OutgoingResponse

Reply:
We already have sigma1/2/3 so it's not a big leap to having sub-steps a/b/c, order is pretty clear. Honestly I find more meaningful names challenging due to the amount of stuff going on in the functions, at least I know where I stand with 1, 2, 3a, 3b, 3c.